js/tests.js

```diff
@@ -52,4 +52,47 @@ describe("get auto-complete context", function() {
         expect(parser.autoCompleteContext('Hi @(CONCAT("@con')).toBeNull();
         expect(parser.autoCompleteContext('Hi @("!" & "@con')).toBeNull();
     });
+
+    it("ignore parenthesis triggering functions completions for variables", function() {
+        expect(parser.autoCompleteContext("Hi @(contact.age")).toBe('contact.age');
+    });
+
+    it('ignore the parenthesis triggering functions completions for functions', function() {
+        expect(parser.autoCompleteContext("Hi @(SUM")).toBe('SUM');
+    });
+
+    it("matches the function without parameters", function() {
+        expect(parser.autoCompleteContext("Hi @(SUM(")).toBe('SUM');
```

Comment:
You need to return something here that indicates to the at.js to display the function summary box rather than the search results for 'SUM'. How about it having it return '#SUM' where the '#' indicates that this is an item match rather than a search?